test/input.txt

```diff
@@ -37,10 +37,11 @@
   add []\[;] p/12345 e/valid@e.mail a/valid, address
   add Valid Name p/not_numbers e/valid@e.mail a/valid, address
   add Valid Name p/12345 e/notAnEmail a/valid, address
+  add Valid Name p/12345 e/invalidEmail@extra.period. a/valid, address
   add Valid Name p/12345 e/valid@e.mail a/valid, address t/invalid_-[.tag
 
   # should add correctly and list non private information
-  add Adam Brown p/111111 e/adam@gmail.com a/111, alpha street
+  add Adam Brown p/1 1 1 1 1 1 e/adam@gmail.com a/111, alpha street
```

Comment:
Do consider additional tests for different whitespaces such as tabs and newline characters.

What about testing the updated regex for emails?

Do note that you were required to update JUnit tests in addition to these I/O tests.